packages/@dcl/inspector/src/components/ImportAsset/Slider/AssetSlides.tsx

```diff
@@ -41,6 +41,19 @@ export function AssetSlides({
   const leftArrowDisabled = currentSlide <= 0
   const rightArrowDisabled = currentSlide >= uploadedAssets.length - 1
 
+  const hasEmoteName = (asset: Asset) => {
+    return asset.name.endsWith('_emote')
+  }
+
+  useEffect(() => {
+    uploadedAssets.forEach((asset, index) => {
+      if (asset.isEmote && !hasEmoteName(asset)) {
+        const newName = `${asset.name}_emote`
+        onNameChange(index)(newName)
+      }
+    })
+  }, [])
```

Comment:
following the same logic from the comment other comment, I think we should do this in `https://github.com/decentraland/js-sdk-toolchain/blob/main/packages/%40dcl/inspector/src/components/ImportAsset/Slider/Slider.tsx#L20` altogether. This way we avoid the `useEffect` usage.